cmd/resource-topology-exporter/main.go

```diff
@@ -112,6 +112,7 @@ func parseArgs(args ...string) (ProgArgs, error) {
 	refCnt := flags.String("reference-container", "", "Reference container, used to learn about the shared cpu pool\n See: https://github.com/kubernetes/kubernetes/issues/102190\n format of spec is namespace/podname/containername.\n Alternatively, you can use the env vars REFERENCE_NAMESPACE, REFERENCE_POD_NAME, REFERENCE_CONTAINER_NAME.")
 
 	flags.StringVar(&pArgs.RTE.NotifyFilePath, "notify-file", "", "Notification file path.")
+	flags.Int64Var(&pArgs.RTE.MaxEventsPerSecond, "max-events-per-second", 1, "Max times per second resources will be scanned and updated")
```

Comment:
the default should be unlimited

Reply:
hm, I asked to change it to 1, to avoid calling a lot of updates by default, IMHO one update per second is more than enough, in general, I preferred more granularity, like work with milliseconds but looks like it is not supported.

Reply:
there is a tradeoff between frequency of updates, thus extra load on the node and most notably on the apiserver, and quality of the scheduling decisions. We need to explore this tradeoff.
I agree that most likely we will need some form of throttling, but this should be a per-cluster decision, thus enforced by either the deployer, the operator, or the cluster admin. Thus, I think the default should be unlimited.

Reply:
Additionally, it's likely we will need to have sub-second granularity (e.g. 100ms)

Reply:
Regarding this ... do you prefer to have a fixed time base and let only the number of events as a configurable value o give the option to configure both of them?